drivers/usb/usbehci_new/usbehci.c

```diff
@@ -0,0 +1,3691 @@
+    { ENDPOINT_INTERRUPT_2ms, 0x00, 0xAA },
+    { ENDPOINT_INTERRUPT_4ms, 0x00, 0x11 },
+    { ENDPOINT_INTERRUPT_4ms, 0x00, 0x44 },
+    { ENDPOINT_INTERRUPT_4ms, 0x00, 0x22 },
+    { ENDPOINT_INTERRUPT_4ms, 0x00, 0x88 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x01 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x10 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x04 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x40 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x02 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x20 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x08 },
+    { ENDPOINT_INTERRUPT_8ms, 0x00, 0x80 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x01 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x01 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x10 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x10 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x04 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x04 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x40 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x40 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x02 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x02 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x20 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x20 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x08 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x08 },
+    { ENDPOINT_INTERRUPT_16ms, 0x01, 0x80 },
+    { ENDPOINT_INTERRUPT_16ms, 0x02, 0x80 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x01 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x01 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x01 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x01 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x10 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x10 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x10 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x10 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x04 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x04 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x04 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x04 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x40 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x40 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x40 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x40 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x02 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x02 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x02 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x02 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x20 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x20 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x20 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x20 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x08 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x08 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x08 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x08 },
+    { ENDPOINT_INTERRUPT_32ms, 0x03, 0x80 },
+    { ENDPOINT_INTERRUPT_32ms, 0x05, 0x80 },
+    { ENDPOINT_INTERRUPT_32ms, 0x04, 0x80 },
+    { ENDPOINT_INTERRUPT_32ms, 0x06, 0x80 },
+    { 0x00, 0x00, 0x00 }
+};
+
+static const UCHAR Balance[EHCI_FRAMES] = {
+    0, 16, 8,  24, 4, 20, 12, 28, 2, 18, 10, 26, 6, 22, 14, 30,
+    1, 17, 9,  25, 5, 21, 13, 29, 3, 19, 11, 27, 7, 23, 15, 31};
```

Comment:
One excess space character can be eliminated, both on lines 97 and 98 (`,  24` and `,  25`).